subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -153,13 +151,37 @@ To use these and similar methods, you need to access the top-level dependencies
 Though it is discouraged, as suggested by the comment in the snippet above, you can also access the configurations of a test suite in a top-level `dependencies` block after creating the test suite.  The configurations used by test suites are normal Gradle configurations, but their exact names should be considered an implementation detail subject to change.  For this reason, configuration via the suite's own `dependencies` block is recommended.
 ====
 
+=== New Strongly-typed Dependencies Block
+The `dependencies` block is using a new API to declare dependencies. This API is very similar to the old API, but it provides full
+typing which makes it easier to understand the relationship between a dependency notation and its `Dependency` sub-type, and
+easier to configure from Kotlin and Java.
```

Comment:
```suggestion
Gradle 8.0 introduced a new dependencies API. This API uses strong
typing which makes it easier to understand the relationship between a dependency notation and its `Dependency` sub-type.
```
Suggestion: Trim some unneeded clauses. Change "new" to "Gradle 8.0 introduced". Use active voice when possible.

Reply:
I think we also shouldn't say "new dependencies API", since I think that oversells the impact of this.  Maybe:

"Gradle 8.0 improved and expanded test suite's dependencies API by using strong typing" or something along those lines.